contracts/lib/LibPerpetuals.sol

```diff
@@ -14,7 +14,7 @@ library Perpetuals {
         uint256 created;
     }
 
-    function orderId(Order calldata order) public pure returns (bytes32) {
+    function orderId(Order memory order) internal pure returns (bytes32) {
```

Comment:
Will we ever want external callers to be able to calculate unique identifiers for orders?